frontend/src/components/public_group/PublicGroupExpensesAndActivity.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+import PublicGroupDonations from './PublicGroupDonations';
+import PublicGroupExpenses from './PublicGroupExpenses';
+
+export default class PublicGroupExpensesAndActivity extends Component {
+  render() {
+    const { group, expenses, donations, users, itemsToShow } = this.props;
+    return (
+      <section id='expenses-and-activity' className='px2'>
+        <div className='container'>
+          <div className='PublicGroup-transactions clearfix md-flex'>
+            <PublicGroupExpenses group={ group } expenses={ expenses } users={ users } itemsToShow={ itemsToShow } {...this.props} />
```

Comment:
`group`, `expenses` etc are already in `this.props` so specifying them explicitly is redundant normally. `<PublicGroupExpenses {...this.props} />` doesn't work?